### PR TITLE
Several updates

### PR DESCRIPTION
**Pull request by SoftCreatR**
- Updated Dockerfile to use Alpine 3.16
- Pinned package versions in Dockerfile
- Dockerfile & entrypoint overhaul

### CODE REVIEW

**Comment by stnguyen90** [comment]
@SoftCreatR Thanks for the PR! 🤯 Please give us some time to review it.

**Comment by stnguyen90** [comment]
`docker buildx` didn't work, but I was able to `docker build` worked on my M1 Mac and on GitHub Codespaces.

**Comment by stnguyen90** [comment]
@TorstenDittmann would you please review?

**Comment by SoftCreatR** [comment]
> `docker buildx` didn't work, but I was able to `docker build` worked on my M1 Mac and on GitHub Codespaces.

Maybe because of this change?

https://github.com/appwrite/docker-telegraf/pull/24/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485L47

File: telegraf.conf [hunk 1]
```diff
@@ -1,138 +0,0 @@
-# Telegraf configuration
```
Comment (TorstenDittmann) [inline telegraf.conf:1]: Why was this removed? 👀

Comment (SoftCreatR) [reply telegraf.conf:1]: Because it's not needed.

**Comment by Meldiron** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

File: telegraf.conf (hunk 1 above)
Comment (christyjacob4) [reply telegraf.conf:1]: @SoftCreatR I'm failing to understand why it is not needed. Could you please explain?

**Comment by christyjacob4** [comment]
@SoftCreatR could you share some screenshots of the built image working in a container?

File: telegraf.conf (hunk 1 above)
Comment (SoftCreatR) [reply telegraf.conf:1]: The default config fits well in this case.

**Comment by SoftCreatR** [comment]
> @SoftCreatR could you share some screenshots of the built image working in a container?

Can't you make it work, or why are you asking?

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)
